BootloaderCommonPkg/Include/Guid/DeviceTableHobGuid.h

```diff
@@ -0,0 +1,47 @@
+/** @file
+  This file defines the hob structure for the device table.
+
+  Copyright (c) 2018, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __DEVICE_TABLE_HOB_GUID_H__
+#define __DEVICE_TABLE_HOB_GUID_H__
+
+///
+/// Device table Hob GUID
+///
+extern EFI_GUID gDeviceTableHobGuid;
+
+typedef struct {
+  UINT8        PciFunctionNumber;
+  UINT8        PciDeviceNumber;
+  UINT8        PciBusNumber;
+  UINT8        IsMmioDevice;
+} SBL_PCI_DEVICE;
+
+typedef struct {
+  UINT8         Type;
+  UINT8         Instance;
+  UINT16        Reserved;
+  union {
+    SBL_PCI_DEVICE  PciDev;
+    UINT32          DevAddr;
+  } Dev;
+} SBL_DEVICE;
+
+#define    SBL_DEVICE_TABLE_SIGNATURE  0xD11D
+typedef struct {
+  UINT16          Signature;
+  UINT16          DeviceNumber;
+  SBL_DEVICE      Device[0];
+} SBL_DEVICE_TABLE;
+
```

Comment:
#pragma pack()

Reply:
already aligned, it is not required. and it is preferred method to avoid to use pack.